src/box/index_def.c

```diff
@@ -94,6 +94,7 @@ const struct opt_def index_opts_reg[] = {
 
 struct index_def *
 index_def_new(uint32_t space_id, uint32_t iid, const char *name,
+	      const char *space_name, const char* engine_name,
 	      uint32_t name_len, enum index_type type,
 	      const struct index_opts *opts,
 	      struct key_def *key_def, struct key_def *pk_def)
```

Comment:
Please keep `name` and `name_len` together in the argument list:

```C
struct index_def *
index_def_new(uint32_t space_id, uint32_t iid,
	      const char *name, uint32_t name_len,
	      const char *space_name, const char *engine_name,
	      enum index_type type, const struct index_opts *opts,
	      struct key_def *key_def, struct key_def *pk_def)
```

Reply:
Fixed.